src/css/base.scss

```diff
@@ -1,10 +1,168 @@
-/* This is an example of how to import a partial scss file*/
 @import '_variables';
 @import '_reset';
 @import '_mixins';
 
+html {
+  @include flex-column();
+  height: 100%;
+}
+
 body {
-  background: url('../images/background-picture.jpg');
-  background-size: cover;
-  background-repeat: no-repeat;
+  @include flex-column();
+  flex: 1;
+  height: 100%;
+  color: $primary-color;
+  font-family: $primary-font;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  &::before {
```

Comment:
I had to create a pseudo element to add opacity to the background picture and prevent it adding the opacity to all text elements within the body.